src/applications/vaos/new-appointment/redux/selectors.js

```diff
@@ -373,7 +373,7 @@ export function getClinicsForChosenFacility(state) {
   return clinics[`${data.vaFacility}_${typeOfCareId}`] || null;
 }
 
-export function getClinicPageInfo(state, pageKey) {
+export const getClinicPageInfo = pageKey => state => {
```

Comment:
I think it's just personal preference, but to me, leaving this selector the way it was and doing `useSelector(state => getClinicPageInfo(state, pageKey), shallowEqual)` is easier to read. Nested arrow functions like this throw me off a little.

Reply:
They throw me off too!  I was trying to force myself to have to read them.  My mother always said I was a glutton for punishment.